thentos-core/src-devel/Paths_thentos_core.hs

```diff
@@ -0,0 +1,6 @@
+-- | data files and development mode.  for details, see:
+-- http://neilmitchell.blogspot.de/2008/02/adding-data-files-using-cabal.html
+module Paths_thentos_core where
+
+getDataFileName :: FilePath -> IO FilePath
+getDataFileName = return
```

Comment:
It is, because nobody knows about this directory except the sensei make rules.

The reason why it is needed is that I create one giant fake package out of all packages that are watched.  The fake package does not have this module, but it is imported somewhere.